src/IROperator.h

```diff
@@ -1636,6 +1636,37 @@ inline Expr count_trailing_zeros(Expr x) {
                                 {x}, Internal::Call::PureIntrinsic);
 }
 
+/** Divide two integers, rounding towards zero. This is the typical
+ * behavior of most hardware architectures, which differs from
+ * Halide's division operator, which is Euclidean (rounds towards
+ * -infinity). */
+inline Expr div_round_to_zero(Expr x, Expr y) {
```

Comment:
Oh, I was thinking in terms of front-end usage of this, but I see that would create an infinite loop in codegen.